source/server/http/admin.cc

```diff
@@ -47,10 +49,75 @@ using namespace rapidjson;
 namespace Envoy {
 namespace Server {
 
+namespace {
+
+/**
+ * Favicon base64 image was harvested by screen-capturing the favicon from a Chrome tab
+ * while visiting https://www.envoyproxy.io/. The resulting PNG was translated to base64
+ * by dropping it into https://www.base64-image.de/ and then pasting the resulting string
+ * below.
+ *
+ * The actual favicon source for that, https://www.envoyproxy.io/img/favicon.ico is nicer
+ * because it's transparent, but is also 67646 bytes, which is annoying to inline. We could
+ * just reference that rather than inlining it, but then the favicon won't work when visiting
+ * the admin page from a network that can't see the internet.
+ */
+const char EnvoyFavicon[] =
```

Comment:
Maybe add this to the style guide, I know it sometimes comes up that `kFooBar` isn't the Envoy way.

Reply:
Added https://github.com/envoyproxy/envoy/pull/2273